source/includes/azure/_instances.md

```diff
@@ -250,6 +250,42 @@ Required | &nbsp;
 ------ | -----------
 `machineType`<br/>*string* | The new machine type to assign to the instance
 
+<!-------------------- RESET PASSWORD -------------------->
+
+#### Reset password
+
+The admin password is used to access Linux or Windows [instances](#azure-instances). 
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/azure/example/instances/subscriptions/subscriptionId/resourceGroups/cmc-example/providers/Microsoft.Compute/virtualMachines/example-small-server?operation=reset_password"
+
+# Request example:
+```
+
+```json
+{
+  "username": "johndoe",
+  "password": "SomePassw0rdVal!d"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id?operation=reset_password</code>
+
+Update the administrator username for the existing instance, or insert a new user.
```

Comment:
```suggestion
Update the administrator username for the existing instance, or insert a new user if it doesn't exist on the instance.
```

And then maybe we can remove the warning?

Reply:
Just refactored the entire paragraph.